packages/http-client-java/generator/http-client-generator-clientcore-test/Generate.ps1

```diff
@@ -102,6 +115,15 @@ if (Test-Path ./src/main/resources/META-INF/client-structure-service_apiview_pro
   Remove-Item ./src/main/resources/META-INF/client-structure-service_apiview_properties.json -Force
 }
 
+# smoke test, generate Java project and verify compilation pass
+git fetch origin pull/6981/head:smoke-test-branch
```

Comment:
Once PR's merged, will `pull/6981/head` still exist?

Reply:
Good question. The head should still exist (even after it get merged). But there could be change to that PR that moves the tsp files or change the folder structure.